<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConversionProgressDialog</class>
 <widget class="QDialog" name="ConversionProgressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MFC Convert Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../gui/gui.qrc">
    <normaloff>:/gui/icons/knut-16.png</normaloff>:/gui/icons/knut-16.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QLabel" name="continueProgressLabel">
     <property name="sizeIncrement">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Continue</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="progressStatusLabel">
     <property name="text">
      <string>Progress status</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QProgressBar" name="conversionProgressBar">
     <property name="value">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="questionIconLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../gui/gui.qrc"/>
 </resources>
 <connections/>
</ui>
